resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.init.erb

```diff
@@ -62,6 +64,11 @@ do_start()
     <%= action %>
     <% end -%>
 
+    if [ ! -d "/var/run/puppetlabs/${REALNAME}" ] ; then
```

Comment:
Only case I can think of that might be a problem maybe is if the user ran the start or foreground script and the permissions got set to root instead of the service user as a result or something.  If you want to just go with what you've got already, though, I'm A-ok with that.